CONTRIBUTING.md

```diff
@@ -0,0 +1,96 @@
+Thank you for your interest in contributing to Glosario!
+We welcome contributions of all kinds.
+If you're here to submit a definition,
+either for a new term 
+or in another language for an existing term,
+please take a moment to read the guidance below.
+
+## 1. Adding a new term to the glossary
+
+To add a new entry, please [fork][forking-guide] the [main Glosario repository][repo]
+and, on a new branch,
+add the term and definition to [`glossary.yml`][glossary].
+This glossary file is written in [YAML].
+Here is an example of how your glossary entry should be structured:
+
+```
+- slug: cran
+  ref:
+    - base_r
+    - tidyverse
+  en:
+    term: "Comprehensive R Archive Network"
+    acronym: "CRAN"
+    def: >
+      A public repository of R [packages](#package).
+```
+
+-   The value associated with the `slug` key identifies the entry.
+    -   It must be unique within the glossary.
+    -   It must be in lower case and use only letters, digits, and the underscore
+        (to be compatible with Jekyll's automatic slug creation).
+-   The entry *may* have a `ref` key.
+    If it is present,
+    its value must be a list of identifiers of related terms in this glossary.
+-   Every other top-level key must be a two-letter ISO 639 language code such as `en` or `fr`.
+  (Refer to the "639-1" column of [this table][iso639-table-en].)
+    -   Every entry must have at least one such language section.
+-   Within each language section for each term:
+    -   The value of `term` is the term being defined.
+        This key must be present.
+    -   The key `acronym` is optional.
+        If present, its value is the acronym for this term.
+    -   The value of `def` is the definition.
+        This key must be present,
+        and the value may contain local links to other terms in this glossary
+        (i.e., links starting with `#`)
+        and/or links to outside sources.
+
+Once your term and definition(s) are complete,
+[make a Pull Request][pr-guide] back to the main repository.
+
+### 2. Adding a definition in another language to an existing term
+
+To add a new definition to an existing term in the glossary,
+please fork [the main Glosario repository][repo]
+and, on a new branch,
+find the term in [`glossary.yml`][glossary] and
+add the two-letter ISO 639 language code such as `en` or `fr`
+as a new top-level key.
+(Refer to the "639-1" column of [this table][iso639-table-en].)
+You can then fill in the `term`, `def`, and (if appropriate) `acronym` for that term in your chosen language.
+See the section above for a full description of the required format for a language term.
+
+## Guide to contributing
```

Comment:
I think the Guide to Contributing should come first since it defines the steps people should go through before contributing.